README.md

```diff
@@ -89,6 +91,47 @@ spec:
 ```
 Above settings will take action on Sat and Sun from 00:00 to 23:59:59, and on Mon-Fri from 00:00 to 08:00 and 20:00 to 23:59:59. If `action:sleep` then runs hibernate at `timeFrom` and unhibernate at `timeTo`.  If `action: delete` then it will delete workloads at `timeFrom` and `timeTo`.
 
+## How to install 
+
+### Using Kubectl 
+
+To install winter soldier, follow the steps given below - 
+
+**STEP 1**
+
+Execute the following command to apply the crd.
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/devtron-labs/winter-soldier/main/config/crd/bases/pincher.devtron.ai_hibernators.yaml 
+```
+
+**STEP 2**
+
+Execute the following command to apply the  winter-soldier controller for installing winter soldier. 
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/devtron-labs/winter-soldier/main/config/hibernator_install.yaml
+```
+
+**STEP 3**
+
+After installing winter soldier, customize or create our own hibernator.yaml template which will contains the included and excluded resources on which winter soldier will be applied.
```

Comment:
After installing winter-soldier, create your own hibernator-policies, you can refer some example policies we have included in <link-to-exmples-folder>